pyxrf/model/fit_spectrum.py

```diff
@@ -1453,7 +1454,11 @@ def fit_per_line_nnls(row_num, data,
 
         result, res = nnls_fit(y, matv, weights=None)
         sst = np.sum((y-np.mean(y))**2)
-        r2_adjusted = 1 - res/(num_data-num_feature-1)/(sst/(num_data-1))
+        if not math.isclose(sst, 0, abs_tol=1e-10):
```

Comment:
Why is it `1e-10` while in the change above it's `1e-20`? Should they be the same?

Reply:
Those changes are in different files and are not directly related. I just picked some small value. I don't know possible range for sst, but I think that it shouldn't go below 1e-10 for reasonable data input. I think we can change it to 1e-20. We don't plot 'sst' at all.

Reply:
OK, thanks for the explanation.